src/main/java/uk/gov/hmcts/darts/audio/component/AudioRequestBeingProcessedFromArchiveQuery.java

```diff
@@ -8,4 +8,5 @@ public interface AudioRequestBeingProcessedFromArchiveQuery {
 
     List<AudioRequestBeingProcessedFromArchiveQueryResult> getResults(Integer mediaRequestId);
 
+    List<AudioRequestBeingProcessedFromArchiveQueryResult> getResultsByMediaIds(List<Integer> mediaIds);
```

Comment:
I think this should be from `hearingId` in a separate query component?

Reply:
created new query component - AudioBeingProcessedFromArchiveQuery